src/renderer/util/fileSystem.js

```diff
@@ -50,7 +51,7 @@ export const moveToRelativeFolder = async (cwd, imagePath, relativeName) => {
   await fs.move(imagePath, dstPath, { overwrite: true })
 
   // dstRelPath: relative directory name + image file name
-  const dstRelPath = path.join(relativeName, path.basename(imagePath))
+  const dstRelPath = normalizePath(path.join(relativeName, path.basename(imagePath)))
   return dstRelPath
```

Comment:
You can simply use `dstRelPath.replace(/\\/g, '/')` because the path is always relative. Remote backward slashes only on Windows because it's a valid Unix path character and then remove the dependency. E.g.

```diff
diff --git a/src/renderer/util/fileSystem.js b/src/renderer/util/fileSystem.js
index c1281ff4..2133930b 100644
--- a/src/renderer/util/fileSystem.js
+++ b/src/renderer/util/fileSystem.js
@@ -6,7 +6,7 @@ import dayjs from 'dayjs'
 import Octokit from '@octokit/rest'
 import { ensureDirSync } from 'common/filesystem'
 import { isImageFile } from 'common/filesystem/paths'
-import { dataURItoBlob } from './index'
+import { isWindows, dataURItoBlob } from './index'
 import axios from '../axios'

 export const create = (pathname, type) => {
@@ -51,6 +51,11 @@ export const moveToRelativeFolder = async (cwd, imagePath, relativeName) => {

   // dstRelPath: relative directory name + image file name
   const dstRelPath = path.join(relativeName, path.basename(imagePath))
+
+  if (isWindows) {
+    // Use forward slashes for better compatibility with websites.
+    return dstRelPath.replace(/\\/g, '/')
+  }
   return dstRelPath
 }
```

Reply:
Thanks for the review.
I'll fix it.